implementations/contracts/ERC725InitAbstract.sol

```diff
@@ -23,35 +16,45 @@ import {OwnableCannotSetZeroAddressAsOwner} from "./errors.sol";
  * @custom:warning This implementation does not have by default a `receive()` or `fallback()` function.
  */
 abstract contract ERC725InitAbstract is
-    Initializable,
-    ERC725XCore,
-    ERC725YCore
+    ERC725XInitAbstract,
+    ERC725YInitAbstract
 {
     /**
      * @dev Internal function to initialize the contract with the provided `initialOwner` as the contract {owner}.
      * @param initialOwner the owner of the contract.
      *
+     * NOTE: we can safely override this function and not call the parent `_initialize(...)` functions from `ERC725XInitAbstract` and `ERC725YInitAbstract`
```

Comment:
Maybe adding a note to say in case changing the logic of _initialize in the child contracts and in case a developer want to have this effect here in ERC725InitAbstract as well, then you should call the specific overrides _initialize function.